runtime/dotnet/azurewebapp/Startup.cs

```diff
@@ -212,5 +209,10 @@ private string GetRootDialog(string folderPath)
 
             throw new Exception($"Can't locate root dialog in {dir.FullName}");
         }
+
+        private bool ConfigSectionValid(string val)
```

Comment:
This could be made static

```suggestion
        private static bool ConfigSectionValid(string val)
```